src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"red", "blue", "pink", "white", "black", "yellow"};
+
+    public String getRandomColor() {
+        Random randomColor = new Random();
```

Comment:
Think about which variables should be local in the method and which should be class-level. In this case, the Random instance `randomColor` could be a private class-level field instead of a method-local variable, as it does not rely on method-specific conditions and can be reused.